src/components/RecentSearchDropdown/RecentSearchDropdown.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { forwardRef, useState } from "react";
+import {
+  StyledClearSpan,
+  StyledListItem,
+  StyledListSpan,
+  StyledRecentSearchDropdown,
+  StyledRemoveIcon,
+  StyledTitle,
+  StyledTitleAndClearContainer,
+  StyledUl,
+} from "./RecentSearchDropdown.style";
+import RemoveIcon from "../../assets/svg/removeIcon.svg";
+import { recentSearchesMock } from "../../mockData/recentSearches";
+
+interface RecentSearchDropdownProps {
+  searches?: string[];
+}
+
+const RecentSearchDropdown: React.ForwardRefRenderFunction<
+  HTMLDivElement,
+  RecentSearchDropdownProps
+> = (props, ref) => {
+  const [recentSearches, setRecentSearches] = useState<string[]>(
+    props.searches || recentSearchesMock
+  );
```

Comment:
You don't need useState.

```suggestion
  const recentSearches = props.searches ?? recentSearchesMock
```